requirements.txt

```diff
@@ -34,3 +34,4 @@ six==1.10.0
 virtualenv==15.1.0
 watchdog==0.8.3
 wrapt==1.10.8
+mysqlclient==1.3.9
```

Comment:
We don't really need multiple database servers, no. Also, the risk is very little for database of our size.